### PR TITLE
Updated galley fetch logic.

### PR DESCRIPTION
**Pull request by ajrbyers**

### CODE REVIEW

**Comment by Bbkctp** [comment]
add to whitelist

File: src/journal/logic.py [hunk 1]
```diff
@@ -63,34 +63,43 @@ def list_scss(journal):
...
+
+def get_galley_content(article, galleys):
+    """
+    Gets the best galley and returns its content
+    :param article: Article object
+    :param galleys: list of Galley objects
+    :return: Inline content of the galley, HTML
+    """
+    galley = get_best_galley(article, galleys)
+    return galley.file_content()
```
Comment (mauromsl) [inline src/journal/logic.py]: `galley` could be `None`

**Comment by mauromsl** [comment]
closes #816

**Comment by ajrbyers** [comment]
And a typo in logic.py line 68 😱

**Pull request merged**